### PR TITLE
LPS-134922 Add API to get or set current account entry

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
No related failure found in ci:test:relevant test suite.

**Comment by drewbrokke** [comment]
ci:reevaluate:1307401_299

File: modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/manager/test/CurrentAccountEntryManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+				TestPropsValues.getGroupId(), user.getUserId()));
+	}
+
+	@Test
+	public void testSetCurrentAccountEntry() throws Exception {
+		AccountEntry accountEntry = AccountEntryTestUtil.addAccountEntry(
+			_accountEntryLocalService);
+
+		_currentAccountEntryManager.setCurrentAccountEntry(
+			accountEntry.getAccountEntryId(), TestPropsValues.getGroupId());
```
Comment (drewbrokke) [inline modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/manager/test/CurrentAccountEntryManagerTest.java:144]: Does this compile? Don't we need the `userId` here?

**Review by drewbrokke — CHANGES_REQUESTED** [review]

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**